packages/block-editor/src/components/block-list/block.js

```diff
@@ -439,7 +429,7 @@ function BlockListBlock( {
 		! showEmptyBlockSideInserter &&
 		! isPartOfMultiSelection &&
 		! isTypingWithinBlock;
-	const shouldShowBreadcrumb = isNavigationMode && isSelected && ! isMultiSelecting;
```

Comment:
This was needed previously to hide the breadcrumb on hover, but this is now gone.